source/extensions/clusters/redis/redis_cluster.h

```diff
@@ -211,6 +211,10 @@ class RedisCluster : public Upstream::BaseDynamicClusterImpl {
     bool enableRedirection() const override { return false; }
     uint32_t maxBufferSizeBeforeFlush() const override { return 0; }
     std::chrono::milliseconds bufferFlushTimeoutInMs() const override { return buffer_timeout_; }
+    std::chrono::milliseconds upstreamDrainPollInterval() const override {
```

Comment:
This is no longer configurable so I this can be removed here and elsewhere.

Reply:
sure enough